src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -3879,6 +3879,29 @@ private void CreateInstanceCheckThis()
             }
         }
 
+        /// <summary>
+        /// Helper to get instances of uninitialized objects.
+        /// </summary>
+        [DebuggerStepThrough]
+        [DebuggerHidden]
+        internal object GetUninitializedObject()
+        {
+            if (GenericCache is not CreateUninitializedCache cache)
+            {
+                if (GenericCache is ActivatorCache activatorCache)
```

Comment:
```suggestion
            object? genericCache = GenericCache;            
            if (genericCache is not CreateUninitializedCache cache)
            {
                if (genericCache is ActivatorCache activatorCache)
```
GenericCache is not a trivial property getter. Avoid calling it twice in a row.